CHANGELOG.md

```diff
@@ -41,6 +41,8 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 - Fixed deserialization of the `MapProof` data structure. (#674)
 
+- Fixed a bug in which the node was unable to reach the actual round. (#680 #681)
```

Comment:
I would say "Fixed a bug which prevented the node from reaching the actual round."